addons/hr_expense/models/hr_expense.py

```diff
@@ -162,7 +162,7 @@ def _default_employee_id(self):
         string="Is currency_id different from the company_currency_id",
         compute='_compute_is_multiple_currency',
     )
-    currency_rate = fields.Float(compute='_compute_currency_rate', digits=(12, 6), readonly=True, tracking=True)
+    currency_rate = fields.Float(compute='_compute_currency_rate', digits=(12, 9), readonly=True, tracking=True)
```

Comment:
```suggestion
    currency_rate = fields.Float(compute='_compute_currency_rate', digits=(16, 9), readonly=True, tracking=True)
```
I would also bump the total digits, not just the decimal part, otherwise you are actually limiting the rate to 999,99...